main.tf

```diff
@@ -103,6 +103,10 @@ resource "google_compute_instance_from_template" "bastion_vm" {
   }
 
   source_instance_template = module.instance_template.self_link
+
+  params {
```

Comment:
Is there any change/direct from always creating the `params {}`?  If so, an option would be to gate using a dynamic block similar to:

```
    dynamic "params" {
      for_each = var.resource_manager_tags != null ? [1] : []
      content {
        resource_manager_tags = var.resource_manager_tags
      }
    }
```

Reply:
@apeabody let me know if you'd like me to implement `params` as a dynamic as you did above, otherwise I think this is good to review.

Reply:
Thanks - sounds like you aren't seeing continuous dirt, so this should be fine.